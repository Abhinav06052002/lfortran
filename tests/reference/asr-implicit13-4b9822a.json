{
    "basename": "asr-implicit13-4b9822a",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/implicit13.f90",
    "infile_hash": "846836e213592204557603a57b6f7420ebd88db79c59d7a369174c36",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-implicit13-4b9822a.stderr",
    "stderr_hash": "4dd5eaa28869f721809250de8371b9acce7ae9fd0f9110a5e5c8e5eb",
    "returncode": 2
}