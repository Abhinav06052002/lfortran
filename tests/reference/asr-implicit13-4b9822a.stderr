semantic error: Implicit typing is not allowed, enable it by using --implicit-typing 
 --> tests/implicit13.f90:2:1
  |
2 | implicit integer (a,b-c), integer*4 (d-e), integer*8 (f-g), real (h)
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
